medicines/pars-upload/src/typography.js

```diff
@@ -1,17 +1,17 @@
-export const H1 = ({ children }) => (
-  <h1 className="govuk-heading-xl">{children}</h1>
+export const H1 = ({ children, component: Component = 'h3' }) => (
```

Comment:
really? everything is an h3?

Reply:
Ah, that is a really dodgy copy and paste error. Good spot